--- ./src/core/web_engine_context.cpp.orig	2017-05-10 14:46:08.946515000 +0800
+++ ./src/core/web_engine_context.cpp	2017-05-11 13:46:03.700011943 +0800
@@ -323,6 +323,8 @@
         parsedCommandLine->AppendSwitchASCII(switches::kProfilerTiming, switches::kProfilerTimingDisabledValue);
     }
 
+    parsedCommandLine->AppendSwitch(switches::kIgnoreGpuBlacklist);
+
     GLContextHelper::initialize();
 
     const char *glType = 0;
